sample-apps/sse/binary/enveloped/consumer.js

```diff
@@ -0,0 +1,25 @@
+(() => {
+    const key = 'API:KEY';
+    const channel = 'sample-app-sse';
+    const url = `https://realtime.ably.io/event-stream?channels=${channel}&v=1.1&key=${key}&delta=vcdiff`;
+    const eventSource = new EventSource(url);
+    const channelDecoder = new DeltaCodec.CheckedVcdiffDecoder();
+    
+    eventSource.onmessage = (event) => {
+        const message = JSON.parse(event.data);

```

Comment:
Could it be worth including some comments describing what form `event.data` takes (i.e. it's a JSON string), as well as where its format is documented?